File: random_probs/two_sum.cpp
#include <iostream>
#include <vector>

using namespace std;

class Solution {
public:
    std::vector<int> twoSum(vector<int>& nums, int target) {
        
        for (int i=0; i<=nums.size(); ++i)
            for (int j=0; j<=nums.size(); ++j)
                if ((nums[j] + nums[i] == target) && (i!=j))
                {
                    return {i,j};
                }
               
    return {};}
};

int main()
{
std::vector<int> sample = {2,5,5,11};
Solution a;
std::vector<int> b = a.twoSum(sample, 10);
cout<<b;
return 0;
}

File: top_interview_quest/medium/two_sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        eligible = []
        for i in range(len(nums)):
            for a,j in enumerate(nums):
                if j<=target:
                    if j+nums[i] ==target and i !=a:
                        return [i,a]